Source/Fuse.Maps/iOS/MapViewDelegate.m

```diff
@@ -214,6 +224,48 @@ -(void) clearMarkers
 			[_mapView removeAnnotation:a];
 		}
 		[_annotations removeAllObjects];
+
+		for(id key in _overlays)
+		{
```

Comment:
Here is the surrounding code, which I copied:

https://github.com/fusetools/fuselibs-public/blob/master/Source/Fuse.Maps/iOS/MapViewDelegate.m#L210-L211

		for(id key in _annotations)
		{

I can make the changes that you requested, but they will be inconsistent with the surrounding code.